code/modules/mechs/interface/datum_HUD.dm

```diff
@@ -9,6 +9,7 @@
 		"mech radio"				=	list("type"	=	/obj/screen/movable/exosuit/radio,				"loc" = "WEST:4,CENTER-3:58"),
 		"rename mech"				=	list("type"	=	/obj/screen/movable/exosuit/rename,				"loc" = "WEST:4,CENTER-3:69"),
 		"mech camera"				=	list("type"	=	/obj/screen/movable/exosuit/toggle/camera,		"loc" = "WEST:4,CENTER-3:80"),
+		"mech power switch"			=	list("type" =	/obj/screen/movable/exosuit/toggle/power_control, "loc" = "WEST:4,CENTER-3:-12"),
```

Comment:
Line this up.

Reply:
it is lined. Check file in view/edit mode.

Reply:
Poggers